packages/zen-bull-vault/src/ZenAuction.sol

```diff
@@ -1,6 +1,5 @@
-// SPDX-License-Identifier: GPL-3.0-only
+//SPDX-License-Identifier: BUSL-1.1
```

Comment:
Imports our ``UniFlash.sol`` and ``UniOracle.sol`` which I think need to be GPL of some kind, but maybe its ok since our contract.

`StrategyMath.sol` is imported here which is `//SPDX-License-Identifier: AGPL-3.0-only` , doesn't this mean we have to make it GPL?